.github/pull_request_template.md

```diff
@@ -54,5 +54,4 @@ Delete this section if no tips.
 - [ ] If this PR evolves [an existing `$T ⇔ Proto$T` mapping](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/command-and-response-binary-encoding.md) (possibly in a backwards-incompatible way), then it is tagged with a `T-proto` label.
 - [ ] If this PR will require changes to cloud orchestration or tests, there is a companion cloud PR to account for those changes that is tagged with the release-blocker label ([example](https://github.com/MaterializeInc/cloud/pull/5021)).
   <!-- Ask in #team-cloud on Slack if you need help preparing the cloud PR. -->
-- [ ] This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):
-  - <!-- Add release notes here or explicitly state that there are no user-facing behavior changes. -->
+- [ ] If this PR includes major [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note), I have scheduled a changelog post.
```

Comment:
> I have scheduled a changelog post.

Can we be more prescriptive here? In the past, it's been great but mostly unfruitful to have Engineers take a stab at writing changelog posts, so we should phrase this differently, _e.g._ ...I have pinged the primary PM to create a Changelog post.

Now that we have more bandwidth in Product and PM assigned to specific product areas, we should explicitly make Changelog within each area the PM's responsibility. cc @JLDLaughlin

Reply:
Sure, works for me!